oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/DocumentPropertyStateTest.java

```diff
@@ -218,22 +218,27 @@ public void stringAboveThresholdSizeNoCompression() {
     public void testInterestingStrings() {
         DocumentNodeStore store = mock(DocumentNodeStore.class);
         String[] tests = new String[] { "simple:foo", "cr:a\n\b", "dquote:a\"b", "bs:a\\b", "euro:a\u201c", "gclef:\uD834\uDD1E",
-                "tab:a\tb", "nul:a\u0000b", "brokensurrogate:\ud800"};
+                "tab:a\tb", "nul:a\u0000b", "brokensurrogate:dfsa\ud800"};
 
         DocumentPropertyState.setCompressionThreshold(1);
         for (String test : tests) {
             if (isValidSurrogatePair(test.split(":")[1])) {
```

Comment:
I'm not sure why we check that here; can't we test all strings?

Reply:
If I test with broken surrogate it will fail, so I filter that value.

Reply:
Done.